.github/workflows/python-package.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
```

Comment:
Maybe rename this to something about lint and test?

Reply:
Sure, renamed file to `python-ci.yaml` and Title to `Python Lint and Test`